frontend/src/views/web/contact-us.html

```diff
@@ -0,0 +1,60 @@
+<div class="white-bg view-container">
+	<main-header></main-header>
+    
+	<div class="grad-container">
+		<div class="row">
+			<div class="col s12 m8 l8">
+				<br><br>
```

Comment:
@Ram81 Don't use br tags like this. Use margin or padding instead of using br tags. Please change accordingly.